db/schema.rb

```diff
@@ -11,15 +11,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160920144648) do
+ActiveRecord::Schema.define(version: 20170316054224) do
```

Comment:
It is the migration that John added the other day. I think he just forgot to update the schema.rb.